docs/rules/README.md

```diff
@@ -40,6 +40,7 @@ The following rules point out areas where you might have made mistakes.
 
 These are rules designed to prevent you from making mistakes. They either prescribe a better way of doing something or help you avoid footguns.
 
+* [accessor-pairs](accessor-pairs.md) - Disallow setters without getters in objects (off by default)
```

Comment:
Enforce getters/setters pairs (we usually use "Disallow" for rules starting with "no-")